### PR TITLE
Added readme for Ephemeral Database action

### PR DESCRIPTION
**Pull request by bricct**
Some TODO's for putting actual yaml examples for the `configuration` input.  Feel free to throw some example config in there if you've got any @anr-tonic.  I know you tend to keep lots of yaml on hand.

### CODE REVIEW

File: ephemeral/database/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+A Tonic Ephemeral host must be provided, alongside a valid API key in order to authenticate with the server.
+
+API keys for a user can be made in the UI of Tonic Ephemeral on the settings page.
+
+# Usage
+
+<!-- start usage -->
+```yaml
+- uses: tonicai/ephemeral-database
```
Comment (anr-tonic) [inline ephemeral/database/README.md:13]:
```suggestion
- uses: tonicai/gha-toolbox/ephemeral/database@main
```

Needs changing in other locations as well.

File: ephemeral/database/README.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+    # The name of the database to either create or destroy
+    name: 'My Ephemeral Database'
+    
+    # Whether to create a database named <name> or to destroy the database named <name>
+    # Possible values: [ 'create-if-not-exists' | 'delete-if-exists' ]
+    mode: 'create-if-not-exists'
+
+    # The configuration to give the database, when in create mode
+    configuration: 'TODO: yaml'
```
Comment (anr-tonic) [inline ephemeral/database/README.md:29]:
```suggestion
    # The configuration to give the database, when in create mode
    # See ephemeral documentation for more details
    configuration: |
      baseImageId: 626ba666-82d7-4466-b885-cdd731834157
      resourceGroupId: 53ef99c8-06ea-4480-a6aa-f4aeabdc6ead
      storageSizeInGigabytes: 5
      volumeReference: example.com/ephemeral-volumes:postgres
      configurationFile: |
        max_connections = 300
        shared_buffers = 80MB
```

File: ephemeral/database/README.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+    name: 'My Ephemeral Database'
+    
+    mode: 'create-if-not-exists'
+
+    configuration: 'TODO: yaml' 
+```
+
+## Destroy Database
+When the mode is specified as `destroy-if-exists` the action will first query the Ephemeral user's databases for a database with the given input `name`.  If exactly one database with a matching name is found, this action will make a request to delete it.
```
Comment (anr-tonic) [inline ephemeral/database/README.md:60]: `destroy-many` is also a valid input; when set `name` is treated as a new line separated collection of database names:

```yaml
- uses: tonicai/gha-toolbox/ephemeral/database@main
  with:
    host: ...
    key: ...
    mode: destroy-many
    name: |
      database1
      database2
```

**Review by anr-tonic — CHANGES_REQUESTED** [review]

File: ephemeral/database/README.md (hunk 3 above)
Comment (bricct) [reply ephemeral/database/README.md:60]: gotcha, saw that in the code but wasn't sure if you wanted to expose that.  Makes sense though and seems useful

**Review by anr-tonic — APPROVED** [review]

**Pull request merged**